www/js/model.js

```diff
@@ -45,6 +49,17 @@ define([
 					};
 				}
 			},
+			setGraphHelpShown: function(_isGraphHelpShown){
+                this.model.properties._isGraphHelpShown = _isGraphHelpShown;
+            },
+            getGraphHelpShown: function(){
+                console.log("model help",this.model.properties);
+                return (this.model.properties._isGraphHelpShown != undefined)?this.model.properties._isGraphHelpShown : false;
+            },
+            setDoneMessageShown : function(_isDoneMessageShown) {
+                this._isDoneMessageShown = _isDoneMessageShown;
+            },
```

Comment:
please use proper indentation.  we use tab which is equal to four spaces. If u use spaces then it is not readable at most of the places.

Reply:
Done!